docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,201 @@
+     ........    ..'.............'..'....      ..........
+   ..'..'...      ...............'.......      ..........
+  ...'......     ...... ..........  ......         .......
+ ...........   .......              ........        ......
+.......        '...'.'.              '.'.'.'         ....
+.......       .....'..               ..'.....
+   ..       ..........               ..'........
+          ............               ..............
+         .............               '..............
+        ...........'..              .'.'............
+       ...............              .'.'.............
+      .............'..               ..'..'...........
+      ...............                 .'..............
+       .........                        ..............
+        .....
+";
+    Console.WriteLine(bot);
+}
+```
+
+### Test the tool
+
+You can run the project and see the output. Try these variations to run our application:
+
+```csharp
+dotnet run
+dotnet run -- "Hello from the bot"
+dotnet run -- hello from the bot
+```
+
+All arguments after the `--` delimiter are passed to our application.
+
+## Setup the global tool
+
+Before you can pack and distribute the application as a Global Tool, you need to modify the project file. Open the `botsay.csproj` file and add three new XML nodes to the `<Project><PropertyGroup>` node:
+
+- \<PackAsTool>  
+Indicates that the application will be packaged for install as a Global Tool.
+
+- \<ToolCommandName>  
+An alternitive name for the tool, otherwise the tool executable will be named after the project file.
+
+- \<PackageOutputPath>  
+Where the NuGet package will be produced. The NuGet package is what the .NET Core CLI Global Tools uses to install your tool.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <PackAsTool>true</PackAsTool>
+    <PackageOutputPath>./nupkg</PackageOutputPath>
+    <ToolCommandName>botsay</ToolCommandName>
+
+  </PropertyGroup>
+
+</Project>
+```
+
+Next, create a NuGet package for your application.
+
+```console
+dotnet pack -c Release
+```
+
+The `botsay.1.0.0.nupkg` file will be created in the folder identified by the `<PackageOutputPath>` XML value from the `botsay.csproj` file, which should be the `./nupkg` folder.
+
+Now that you have a package, install the tool from that package: 
+
+```console
+dotnet tool install -g --add-source ./nupkg botsay`
+```
+
+The `--add-source` parameter tells .NET Core to use the `./nupkg` folder (our `<PackageOutputPath>` folder) as an additional feed source for NuGet packages. For more information about installing Global Tools, see [.NET Core Global Tools overview][global-tool-info].
+
+> [!NOTE]
+> If the version (default is 1.0.0) of your application is lower than 1.0.0, say 0.1, you will need to specify the `--version 0.1` parameter on the `dotnet tool install` command.
```

Comment:
@wli3 Do you need this for 0.1? Do we still need it for previews?

@Thraka You need to specify the version when the consumer wants is not the latest?

Reply:
I'll clarify this and point to the nuget versioning page which talks about what is considered "preview"

Reply:
@KathleenDollard Actually, I'm going to remove this note as it feels like unnecessary noise. If they follow the steps for this article, their version is 1.0.0.